stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -112,6 +112,7 @@ public static Source createSourceOnServer(
      * @throws APIConnectionException if there is a problem connecting to the Stripe API
      * @throws APIException for unknown Stripe API errors. These should be rare.
      */
+    @Deprecated
```

Comment:
So can/do people use `StripeApiHandler` rather than `Stripe` to make requests? Not sure if this is in your plans for 5.0, but maybe it's worth deprecating the class and eventually making it private?